bin/kong

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env lua
+#!/usr/bin/env luajit
```

Comment:
I am just concerned that some people might not have luajit in their PATH (only lua), and that this might break the CLI for them. We'll need to add a breaking change note to the changelog.